src/Pelagos/Entity/DatasetSubmission.php

```diff
@@ -1004,6 +1013,7 @@ public function __construct(Entity $entity, PersonDatasetSubmissionDatasetContac
             $this->setTemporalExtentNilReasonType($entity->getTemporalExtentNilReasonType());
             $this->setDistributionFormatName($entity->getDistributionFormatName());
             $this->setFileDecompressionTechnique($entity->getFileDecompressionTechnique());
+            $this->setErddapUrl($entity->getErddapUrl());
```

Comment:
It is debatable if this is needed, because the URL could get invalidated after another submissions. Maybe a questions for the PO?

Reply:
The only way to set this field is through Dataset-review tool, and if we don't add it in here, the data from a previous submission would be lost. For example: If dataset in accepted status, with an existing erddapUrl is opened in dataset-review tool, the new object would have an empty erddapUrl attribute.

Reply:
Right again!